airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -695,39 +1092,79 @@ definitions:
               - GET
               - POST
         default: GET
+        examples:
+          - GET
+          - POST
       request_body_data:
+        title: Request Body Payload (Non-JSON)
         description: Specifies how to populate the body of the request with a non-JSON payload. If returns a ready text that it will be sent as is. If returns a dict that it will be converted to a urlencoded form.
         anyOf:
           - type: string
           - type: object
             additionalProperties:
               type: string
+        interpolation_context:
+          - next_page_token
+          - stream_slice
+          - stream_state
+        examples:
+          - |
+            [{"clause": {"type": "timestamp", "operator": 10, "parameters":
+                [{"value": {{ stream_slice['start_time'] | int * 1000 }} }]
+              }, "orderBy": 1, "columnName": "Timestamp"}]/
       request_body_json:
+        title: Request Body JSON Payload
         description: Specifies how to populate the body of the request with a JSON payload.
         anyOf:
           - type: string
           - type: object
             additionalProperties:
               type: string
+        interpolation_context:
+          - next_page_token
+          - stream_slice
+          - stream_state
+        examples:
+          - sort_order: "ASC"
+            sort_field: "CREATED_AT"
+          - key: "{{ config['value'] }}"
       request_headers:
+        title: Request Headers
         description: Return any non-auth headers. Authentication headers will overwrite any overlapping headers returned from this method.
         anyOf:
           - type: string
           - type: object
             additionalProperties:
               type: string
+        interpolation_context:
+          - next_page_token
+          - stream_slice
+          - stream_state
+        examples:
+          - Output-Format: JSON
+          - Version: "{{ config['version'] }}"
       request_parameters:
+        title: Query Parameters
         description: Specifies the query parameters that should be set on an outgoing HTTP request given the inputs.
         anyOf:
           - type: string
           - type: object
             additionalProperties:
               type: string
+        interpolation_context:
+          - next_page_token
+          - stream_slice
+          - stream_state
+        examples:
+          - unit: "day"
+          - query: 'last_event_time BETWEEN TIMESTAMP "{{ stream_slice.start_time }}" AND TIMESTAMP "{{ stream_slice.end_time }}"'
+          - searchIn: "{{ ','.join(config.get('search_in', [])) }}"
+          - sort_by[asc]: updated_at
       $parameters:
         type: object
         additionalProperties: true
   HttpResponseFilter:
-    description: A filter that is used to select on properties of the HTTP response received
+    description: A filter that is used to select on properties of the HTTP response received.
```

Comment:
Could we add here whether the different filters (message contains/status code/predicate) are AND-ed or OR-ed?

Reply:
done